src/libraries/Native/Windows/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -123,4 +122,6 @@ if (GEN_SHARED_LIB)
 endif()
 
 install (TARGETS System.IO.Compression.Native-static DESTINATION ${STATIC_LIB_DESTINATION})
-
+if(WIN32)
+    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/System.IO.Compression.Native-static.dir/$<CONFIG>/System.IO.Compression.Native-static.pdb DESTINATION ${STATIC_LIB_DESTINATION})
```

Comment:
We need to binplace the .pdb as well for the lib to be actually useful. It is similar to what we do in other places.

Reply:
Comment in `strip_symbols` mentions that `$<TARGET_PDB_FILE:tgt>` could be used, but it does not work with resource dlls. 
This is not a resource dll, so maybe  `TARGET_PDB_FILE` would work here?

Reply:
hmm, it looks like `TARGET_PDB_FILE` does not work with static libs either. 
https://gitlab.kitware.com/cmake/cmake/-/issues/16932

Reply:
I am going to omit this change from this PR.